### DISCUSSION TITLE
ESPHome not working with cloudflared

### DISCUSSION THREAD

**Discussion by feinadam** [homeassistant-apps/app-cloudflared#483] [Q&A] [answered]
Hello!

Since I have installed the addon my ESP home device is not working.
I can have access to the ESPhome device even OTA is working. I have no acces to the sensors of the esphome device. I have receiving the following error in the log:

INFO Starting log output from 192.168.1.90 using esphome API
WARNING Can't connect to ESPHome API for 192.168.1.90: Error connecting to ('192.168.1.90', 6053): [Errno 113] Connect call failed ('192.168.1.90', 6053) (SocketAPIError)
INFO Trying to connect to 192.168.1.90 in the background

Without the cloudflare the same esphome configuration is working well.

Do you have any idea?

Thank you in advance!

**Comment by brenner-tobias** [comment]
This is very strange. The addon does not change how HA is available within your own network. Now since I assume the ESPHome devices are all communicating locally, the addon should not interfere at all.

Can you please confirm one thing: Check that everything is working without the addon. Install and configure the addon and check if it is still working (via browsers and IP of HA). If it is not working, simply stop the addon and check again.

@elcajon, @AlecRust @felipecrs does anyone of you use ESPHome and might have some insights here?

**Reply by felipecrs** [reply]
That's strange indeed. I use ESPHome add-on with eight ESPHome devices, and I even frequently update them. And I have always used Cloudflared too. No issues whatsoever.

**Reply by felipecrs** [reply]
@feinadam Just to confirm, the minute you _STOP_ the Cloudflared add-on, ESPHome starts working again?

**Comment by feinadam** [comment, marked as answer]
Hello everybody!

Thank you for the hints.
The issue was not coused by cloudflare addon.
It was in the sourcecode of the esphome device.
Unfortunately only in very special case could it happen. Therefore i wrongly suspected for cloudflare extension.
